webapp/packages/plugin-authentication-administration/src/Administration/Users/UsersTable/CreateUserBootstrap.ts

```diff
@@ -28,18 +29,28 @@ export class CreateUserBootstrap extends Bootstrap {
     super();
   }
 
-  override register() {
+  override register(): void {
     this.menuService.addCreator({
       menus: [MENU_USERS_ADMINISTRATION],
       getItems(context, items) {
-        return [...items, ACTION_CREATE];
+        return [...items, ACTION_CREATE, ACTION_REFRESH];
+      },
+      orderItems(context, items) {
+        const refreshIndex = items.indexOf(ACTION_REFRESH);
+
+        if (refreshIndex > -1) {
+          const [refresh] = items.splice(refreshIndex, 1);
+          items.push(refresh!);
+        }
```

Comment:
there is special helpers: `menuExtractItems`, `menuFindIndex` and `menuItemsPlaceAfter`

in this case you probably want:
```
items.push(...menuExtractItems(items, [ACTION_REFRESH]));
```